src/global/Constants.js

```diff
@@ -57,3 +57,7 @@ export const NATIONS_PROD_ENDPOINT = 'https://nation-cache.herokuapp.com/nations
  * @type {number}
  */
 export const CHAT_MESSAGES_PAGE = 20;
+
+// Transaction Values
+
+export const DEFAULT_GAS_LIMIT = '21000';
```

Comment:
will the default always be 21000? 21K is just enough for a normal eth tx.

Reply:
Yes, it's good enough for normal transactions

Reply:
and what is about contract defaults?

Reply:
* I mean token defaults

Reply:
For tokens will be calculated on real time (as for all transactions) but we needed to stablished a default Gas Limit to be the base

Reply:
ok